src/NServiceBus.Core/Licensing/Browser.cs

```diff
@@ -1,30 +1,26 @@
-﻿namespace NServiceBus
+﻿#if NETCOREAPP2_0
+namespace NServiceBus
 {
     using System;
-
-#if NETCOREAPP2_0
     using System.Diagnostics;
     using System.Runtime.InteropServices;
-#endif
 
     static class Browser
     {
-
-        // taken from: https://brockallen.com/2016/09/24/process-start-for-urls-on-net-core/
-        public static void OpenBrowser(string url)
+        public static void Open(string url)
         {
-#if NETCOREAPP2_0
             try
             {
-                Process.Start(url);
```

Comment:
This doesn't really do anything on .NET Core except throw an exception in all cases. On Windows, you have to manually set `UseShellExecute` to true (unlike on the framework), and you have to use the open commands on mac or linux.